Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-alpine
+
+RUN apt-get update && apt-get install -y git && apt-get clean
+
+WORKDIR /app
+
+RUN git clone https://github.com/getsentry/usage-accountant.git .
```

Comment:
You are not supposed to need git in your Dockerfile, this is already executed from the root of your repo.

More importantly I don't see you installing the dependencies at any point, nor have an EXEC or CMD directive which would tell the container what to run. I am not sure this would work.